.github/workflows/build.yml

```diff
@@ -150,15 +150,64 @@ jobs:
     - name: Install cht-conf
       run:  npm install -g cht-conf
     - run: npm ci --legacy-peer-deps
+# TODO only run on "master" and "push"
+    - name: Fetch allure history
```

Comment:
Right. I don't want to get distracted by someone making a change in a branch that actually breaks a test. Also allure works on execution timelines so it would be really confusing if running a 3.x build or something...

Reply:
what i was referring to was needing to download all allure history locally and then force push it back.

Reply:
Ahh, right, yes!